salt/base/swap.sls

```diff
@@ -0,0 +1,16 @@
+{% set swapfile = salt["pillar.get"]("swapfile", {}) %}
+{% set size = swapfile.get("size", "1024") %}
+{% set path = swapfile.get("path", "/swapfile") %}
+
+{{ path }}:
```

Comment:
Given suggested change above, wrapping this in a `{% if path is not None %}` would make this opt-in.